framework/console/Request.php

```diff
@@ -71,6 +71,11 @@ public function resolve()
                 if ($name !== Application::OPTION_APPCONFIG) {
                     $params[$name] = isset($matches[3]) ? $matches[3] : true;
                 }
+            } elseif (preg_match('/^-(\w+)(=(.*))?$/', $param, $matches)) {
```

Comment:
Ok, It should be applied for this (https://github.com/pana1990/yii2/blob/10840-add-shorcuts-console/framework/console/Request.php#L69) too ?